spec/controllers/conversations_controller_spec.rb

```diff
@@ -102,6 +102,14 @@
       get :index, conversation_id: @conversations.first.id
       expect(assigns[:conversation]).to be_nil
     end
+
+    it "retrieves a conversation message with out markdown content " do
+      get :index
+      @conversation = @conversations.first
+      expect(response).to be_success
+      expect(response.body).to match(%r{cool stuff})
```

Comment:
Please refer [Rubocop](http://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/RegexpLiteral) as it suggest to use either `%r{}` or `//`